bin/release_describer

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env ruby
+# Usage:
+# The first argument is the path to the directory containing the repos
+# The second argument is the previous release tag
+# bin/release_describer ~/Workspace v2023-11-08
+
+require "set"
+
+repos = ["nucore-open", "nucore-nu", "nucore-osu", "nucore-umass", "nucore-dartmouth"]
+repo_dir = ARGV[0]
+previous_release_tag = ARGV[1]
+
+data = Hash.new do |h, k|
+  h[k] = {
+    shared_dev: Set.new,
+    open_highlights: Set.new,
+    fixes: Set.new,
+    tech_tasks: Set.new,
+    security: Set.new,
+    school_specific: Set.new,
+    behind: Set.new,
+    ahead: Set.new,
+  }
+end
+repos.each do |repo|
+  puts "Processing changes from #{repo} ..."
+  git_log = `cd #{repo_dir}/#{repo} && bin/merge_describer #{previous_release_tag}`
+  git_log.split("\n").each do |line|
+    ticket_number_match = line.match(/\[#(\d*)/)
+    formatted_line = if ticket_number_match
+      line.gsub(/\[#(\d*)\]/, "[##{ticket_number_match[1]}](https://pm.tablexi.com/issues/#{ticket_number_match[1]})")
+    else
+      line
+    end
+
+    if line.include?("nucore-open")
+      if line.include?("Shared Dev")
+        data[repo][:shared_dev] << formatted_line
+      elsif line.start_with?("* Bump") || line.include?("Tech task")
+        data[repo][:tech_tasks] << formatted_line
+      elsif line =~ /\bSecurity:/i
```

Comment:
```suggestion
      elsif line.downcase =~ /\bsecurity:/i
```

Reply:
The `i` at the end makes this a case insensitive regex - but maybe I'll stick with something more semantic like:
`line.downcase.start_with?("Security:")`